kolibri/plugins/learn/templates/learn/learn.html

```diff
@@ -1,18 +1,19 @@
 {% extends "kolibri/base.html" %}
 {% load learn_tags %}
+{% load kolibri_tags %}
 
 {% block content %}
+{% kolibri_bootstrap_collection 'channel' 'ChannelResource' %}
   <script>
```

Comment:
My intuition is that rather than having a special case for this particular model, it would be better to do it consistently.

It might actually be possible to do through the existing API - is there an API version of this lookup?

```python
ContentNode.objects.get(parent__isnull=True)
```

If not, we could possibly add a new API call specifically for the root node.

(cc @jamalex )